### PR TITLE
m_verifivation_failed=>m_verification_failed

### PR DESCRIPTION
**Pull request by RocksteadyTC**
typo

### CODE REVIEW

**Comment by FndNur1Labs** [comment]
why split. not all PR. but nice keep good work~

**Comment by RocksteadyTC** [comment]
@FndNur1Labs Thanks :smile_cat:

**Comment by FndNur1Labs** [comment]
maybe the owner is drunk then he not put that typo xD

File: tests/UnitTests/TransactionPool.cpp [hunk 1]
```diff
@@ -261,13 +261,13 @@ TEST_F(tx_pool, double_spend_tx)
...
   ASSERT_TRUE(test.pool.add_tx(tx, tvc, false));
-  ASSERT_FALSE(tvc.m_verifivation_failed);
+  ASSERT_FALSE(tvc.m_verification_failed);
 
   test.txGenerator.rv_acc.generate(); // generate new receiver address
   test.construct(test.m_currency.minimumFee(), 1, tx_double);
 
   ASSERT_FALSE(test.pool.add_tx(tx_double, tvc, false));
-  ASSERT_TRUE(tvc.m_verifivation_failed);
+  ASSERT_TRUE(tvc.m_verification_failedd);
```
Comment (LouisWayne) [inline tests/UnitTests/TransactionPool.cpp:270]: Typo here

**Pull request closed without merging**